.github/workflows/node.js.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
did you really intend to test across all three version of Node?

Reply:
Sorry for the late reply! I didn't intend to, but I was having an issue when working on it, and it seemed adding additional versions helped